subsys/bluetooth/host/Kconfig

```
@@ -632,9 +632,14 @@ config BT_BONDABLE
 	bool "Bondable Mode"
 	default y
 	help
-	  This option enables support for Bondable Mode. In this mode,
-	  Bonding flag in AuthReq of SMP Pairing Request/Response will be set
-	  indicating the support for this mode.
+	  This option enables support for Bondable Mode. This option is the
+	  default value of the bonding flag for any ACL connection. If the option
```

```suggestion
	  This option is the
	  default value of the bonding flag for any ACL connection. If the option
```

Isn't this more correct? As far as I can tell, a device can still enter the bondable mode even if this is disabled, so this option does not control the bondable mode, but only what the initial bondable state is. 

Arguably, since the bondable state can be changed at runtime, we should just remove this option. If we reduce it to just control the initial value of `bondable`, then it's pretty useless alongside `bt_set_bondable`. 

If we actually want to control whether bonding is support or not by this Kconfig, that's apparently a larger change.

---

Yes, But if this config is removed, how to control he initial value of `bondable`?

> Arguably, since the bondable state can be changed at runtime, we should just remove this option. If we reduce it to just control the initial value of `bondable`, then it's pretty useless alongside `bt_set_bondable`.

---

The sane default is true, as per the Kconfig default. That said, this is a breaking change which needs to be documented. Luckily, removing the option will cause a build failure for any out of tree app that so far tried to disable it, so it won't go silently unnoticed.